### PR TITLE
Simplex payment

### PR DESCRIPTION
**Pull request by michal-stybowski-ft**

### CODE REVIEW

**Comment by michal-stybowski-ft** [comment]
please, going forward never create such big pull requests. @NikitaFedorov1. This is disrupting our workflow.

**Review by michal-stybowski-ft — COMMENTED** [review]
For such a large PR some description is highly advisable. Write about the purpose of the changes, what it introduces and how to test that. If possible link that to appropriate linear tasks.

File: lib/env/structs.ts [hunk 1]
```diff
@@ -70,6 +85,23 @@ export function underTheFlag<TFlag extends string, S extends ObjectSchema>(
   ]);
 }
 
+/**
```
[the remark this answers is not in the document]
Comment (NikitaFedorov1) [reply lib/env/structs.ts:88]: Keeping documentation close to the code helps maintain it alongside the code. When the code changes, it is easier to remember to update the documentation, ensuring it stays accurate and up-to-date. When developers are working within the codebase, having the comment immediately available provides quick access to essential information about the function without needing to switch contexts or navigate away from the code. This immediacy is crucial during development and debugging.

On the other hand README offers a broader overview of the project, including installation instructions, overall architecture, general usage, and high-level features. It is intended for new users to get started with the project and understand its overall purpose and structure. It requires navigation away from the code to access detailed information about specific functions.

File: integrations/wert/src/api/wertSupportedFiatCurrencies.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+// from widget, but should be from https://sandbox.wert.io/api/v3/pairs?is_sc=false
```
[the remark this answers is not in the document]
Comment (NikitaFedorov1) [reply integrations/wert/src/api/wertSupportedFiatCurrencies.ts:1]: That's why I haven't changed it yet.

File: integrations/wert/src/api/wertSupportedCryptoCurrencies.ts
[the remark this answers is not in the document]
Comment (NikitaFedorov1) [reply integrations/wert/src/api/wertSupportedCryptoCurrencies.ts:1]: This is still needed for compatibility checks and to run tests without listing the tico tokens.

**Review by NikitaFedorov1 — APPROVED** [review]

**Pull request merged**